packages/dd-trace/test/appsec/next/pages-dir/server.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line strict
```

Comment:
why disable the rule if it's here anyway ?

Reply:
For some reason, ESLint is complaining that there shouldn't be a need to explicitly enable strict mode using `'use strict'` below. I can't figure out why it thinks that, but I believe it's wrong. So I have chosen to ignore the error using this comment.

Alternatively I remove the line below, but as I said, I think it's actually required. If you know why, please let me know